Functions/Verbs/msvc90/script.js

```diff
@@ -0,0 +1,25 @@
+include(["Functions", "Engines", "Wine"]);
+include(["Functions", "Net", "Resource"]);
+include(["Functions", "Filesystem", "Extract"]);
+include(["Functions", "Verbs", "luna"]);
+
+Wine.prototype.msvc90 = function() {
```

Comment:
I know. The strange thing is: I tried Quantum Conundrum with vcrun2008 and it didn't work. With the files from msvc90 it worked. No idea why.